settingshelper.py

```diff
@@ -192,8 +193,8 @@ def fix_logger_obfuscation(fix_logger_obfuscation_, logging_config):
         for handler in logging_config["handlers"].values():
             if handler["class"].startswith("corehq."):
                 if fix_logger_obfuscation_ != 'quiet':
-                    print "{} logger is being changed to {}".format(
+                    print("{} logger is being changed to {}".format(
                         handler['class'],
                         'logging.StreamHandler'
-                    )
+                    ), file=sys.stderr)
```

Comment:
It was printing stuff to stdout when I ran `./manage.py list_couchdbs` and that was preventing me from using the output of that command to specify the dbs for the other command.